db/migrate/20230609095923_keyword.rb

```diff
@@ -0,0 +1,8 @@
+class Keyword < ActiveRecord::Migration[7.0]
+  def change
+    create_table :keywords do |t|
+      t.string :name
```

Comment:
```suggestion
      t.string :name, index: true
```

As we will perform `search` on the keyboard names, it is nice to add an `index` on it.

In a simplified verison, the arbitration for indexes is:
- is update/write speed critical? if yes, avoid indexes (as each value change will require the DB to rebuild the index)
- is read/retrieval/search/ordering critical? if yes, you probably need an index